src/test/appVersions.test.js

```diff
@@ -208,7 +208,7 @@ describe("App Version List component", function () {
 describe("App Version component", function () {
 
   beforeEach(function () {
-    var model = {
+    this.model = {
```

Comment:
Just changed it, because I need it here:
https://github.com/mesosphere/marathon-ui/pull/171/files#diff-7b62505cd99e9b6f94661bf24b61fa77R253